doc/articles/features/windows-storage-pickers.md

```diff
@@ -30,6 +30,30 @@ On some platforms, you can further customize the file picking experience by util
 
 On platforms where the additional features are not supported yet, setting them will not have any effect.
 
+## Uno.WinUI-specific Initialization
+
+If you are using Uno.WinUI (Windows App SDK-based form of Uno Platform), you need to make sure the pickers are initialized for the current window, otherwise the code would fail on Windows. This is done using `WinRT.Interop` and is required for all file and folder pickers. For example in casde of `FileOpenPicker`:
```

Comment:
```suggestion
If you are using Uno.WinUI (Windows App SDK-based form of Uno Platform), you need to make sure the pickers are initialized for the current window, otherwise the code would fail on Windows. This is done using `WinRT.Interop` and is required for all file and folder pickers. For example, in case of `FileOpenPicker`:
```